### PR TITLE
Fix/entrypoints action

### PR DESCRIPTION
**Pull request by gwyneplaine**
## DTOK-36

[Jira Issue Link](https://product-fabric.atlassian.net/jira/software/projects/DTOK/boards/371?selectedIssue=DTOK-36)

Brief issue description here.
### General 
- remove rimraf in favor of simpler fs procedures. 
### Better Undo Method
- now only deletes directories and associated files created by do method, as oppose to deleting everything in design-system-js.

### Checklist
- [x] Jira issue status updated
- [x] Tests added
- [x] Reviewers tagged

### Follow-up actions 
- [Refactor File IO](https://product-fabric.atlassian.net/browse/DTOK-65) to be more declarative and async

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -35,11 +35,9 @@ function createFiles(platform) {
   // first do a rollup of all tokens
   let files = formats[platform].map((format) => createFile(format, 'index'));
 
-  // console.log(files);
-
   // then, for each category, output a file in each format
   Object.keys(properties).forEach(function(category) {
-    let filter = (prop) => prop.attributes.category == category;
+    let filter = (prop) => prop.attributes.category === category;
```
Comment (gwyneplaine) [inline build.js:40]: Boyscouting! Didn't see a reason for loose equality here.
Generally I prefer strict equality over loose equality unless there's an explicit reason for it. 

If there is a reason though, happy to revert this change!

**Comment by gwyneplaine** [comment]
@sarahbethfederman would be good to get both this and #43 reviewed and merged in by the end of the week so we can start moving everything to https://github.com/atlassian/design-tokens

File: build.js (hunk 1 above)
Comment (sarahbethfederman) [reply build.js:40]: No reason, just an oversight 👍 good catch

**Review by sarahbethfederman — APPROVED** [review]
LGTM, thanks for taking this on!
